Configurations/ConfigCommon.xcconfig

```diff
@@ -20,9 +20,10 @@ SPARKLE_ICON_NAME = AppIcon
 SPARKLE_VERSION_MAJOR = 2
 SPARKLE_VERSION_MINOR = 0
 SPARKLE_VERSION_PATCH = 0
-SPARKLE_VERSION = $(SPARKLE_VERSION_MAJOR).$(SPARKLE_VERSION_MINOR).$(SPARKLE_VERSION_PATCH)
+SPARKLE_VERSION_SUFFIX =
```

Comment:
let's add a comment illustrating what the expected format of this should be when not empty (eg -beta.1)

Reply:
also a comment that if we change the major/minor/patch/suffix, we should bump the project version.

Reply:
Done in 9cff8f9f5cf6007a33e38dda6954331678f7241c.